packages/json/src/checker/index.ts

```diff
@@ -2,6 +2,109 @@ import type * as core from '@spyglassmc/core'
 import * as mcdoc from '@spyglassmc/mcdoc'
 import { type JsonNode, JsonPairNode } from '../node/index.js'
 
+const Options: (
+	ctx: core.CheckerContext,
+) => mcdoc.runtime.checker.McdocCheckerOptions<JsonNode> = (ctx) => ({
```

Comment:
This is a function, so it should be called `function getOptions(ctx: core.CheckerContext)`

Reply:
Gotcha, I'll take care of that later today